### DISCUSSION TITLE
How do I the contact email (not public) of the individual sponsors?

### DISCUSSION THREAD

**Discussion by RaenonX** [community/community#75344] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Searched through the API doc but I couldn't find it. My sponsoring perks include digital access to certain stuff, so I have to send them the accessing link individually. The current "Send Email Update" won't work, because it's sending the email to everyone.

The GraphQL of the below appears to be grabbing public email address.
```graphql
query {
  user(login:"raenonx") {
    ... on Sponsorable {
      sponsors(first: 10) {
        totalCount
        nodes {
          ... on User { 
            login,
            email,
          }
          ... on Organization { 
            login
          }
        }
      }
    }
  }
}
```

**Comment by b8zeek** [comment, marked as answer]
There is no way to access a sponsor's private email address! You shouldn't even try to find one because it doesn't exist 100%! The email field in the GraphQL API only returns the publicly visible email address set in the user's profile, you should satisfy yourself with that.

Alternative and I would say the right solution for you problem is maybe to update your sponsorship process by letting your sponsors know about those perks. You can ask them to provide their email addresses during the sponsorship process, maybe something like asking them to email you after they sponsor you, or you could set up a form that they should fill out where you would collect the addresses.

And make sure to be careful with someone's privacy, people find that really important.

Hope this helps. 🍺 

🍀

**Comment by RaenonX** [comment]
I see, thanks!

**Reply by b8zeek** [reply]
Let me know if this above helped and how you resolved the issue.

🍀

**Reply by RaenonX** [reply]
Ends up deciding to ask the subscribers to contact me directly, I won't say it helped because I want to automate the activation process, but it does help by making me to think of an alternative not-that-great manual way

**Reply by b8zeek** [reply]
Create a form which they can fill during the registration process and collect the data that way. And make sure to properly inform them about the perks, I would appreciate being informed about that. :)

🍀
